src/lib.rs

```diff
@@ -539,128 +558,301 @@ macro_rules! define_sentence_type_enum {
-    /// General: OSD |
-    /// Autopilot: APA | APB | ASD |
-    /// Decca: DCN |
-    /// D-GPS: MSK
-    /// Echo: DBK | DBS | DBT |
-    /// Radio: FSI | SFI | TLL
-    /// Speed: VBW | VHW | VLW |
-    /// GPS: ALM | GBS | GGA | GNS | GSA | GSV |
-    /// Course: DPT | HDG | HDM | HDT | HSC | ROT | VDR |
-    /// Loran-C: GLC | LCD |
-    /// Machine: RPM |
-    /// Navigation: RMA | RMB | RMC |
-    /// Omega: OLN |
-    /// Position: GLL | DTM
-    /// Radar: RSD | TLL | TTM |
-    /// Rudder: RSA |
-    /// Temperature: MTW |
-    /// Transit: GXA | RTF |
-    /// Waypoints and tacks: AAM | BEC | BOD | BWC | BWR | BWW | ROO | RTE |
-    ///                      VTG | WCV | WNC | WPL | XDR | XTE | XTR |
-    /// Wind: MWV | VPW | VWR |
-    /// Date and Time: GDT | ZDA | ZFO | ZTG |
+    ///
+    /// ## Types
+    ///
+    /// ### General
+    ///
+    /// - [`SentenceType::OSD`]
+    ///
+    /// ### Autopilot:
+    ///
+    /// - [`SentenceType::APA`]
+    /// - [`SentenceType::APB`]
+    /// - [`SentenceType::ASD`]
+    ///
+    /// ### Decca
+    ///
+    /// - [`SentenceType::DCN`]
+    ///
+    /// ### D-GPS
+    ///
+    /// - [`SentenceType::MSK`]
+    ///
+    /// ### Echo
+    /// - [`SentenceType::DBK`]
+    /// - [`SentenceType::DBS`]
+    /// - [`SentenceType::DBT`]
+    ///
+    /// ### Radio
+    ///
+    /// - [`SentenceType::FSI`]
+    /// - [`SentenceType::SFI`]
+    /// - [`SentenceType::TLL`]
+    ///
+    /// ### Speed
+    ///
+    /// - [`SentenceType::VBW`]
+    /// - [`SentenceType::VHW`]
+    /// - [`SentenceType::VLW`]
+    ///
+    /// ### GPS
+    ///
+    /// - [`SentenceType::ALM`]
+    /// - [`SentenceType::GBS`]
+    /// - [`SentenceType::GGA`]
+    /// - [`SentenceType::GNS`]
+    /// - [`SentenceType::GSA`]
+    /// - [`SentenceType::GSV`]
+    ///
+    /// ### Course
+    ///
+    /// - [`SentenceType::DPT`]
+    /// - [`SentenceType::HDG`]
+    /// - [`SentenceType::HDM`]
+    /// - [`SentenceType::HDT`]
+    /// - [`SentenceType::HSC`]
+    /// - [`SentenceType::ROT`]
+    /// - [`SentenceType::VDR`]
+    ///
+    /// ### Loran-C
+    ///
+    /// - [`SentenceType::GLC`]
+    /// - [`SentenceType::LCD`]
+    ///
+    /// ### Machine
+    ///
+    /// - [`SentenceType::RPM`]
+    ///
+    /// ### Navigation
+    ///
+    /// - [`SentenceType::RMA`]
+    /// - [`SentenceType::RMB`]
+    /// - [`SentenceType::RMC`]
+    ///
+    /// ### Omega
+    ///
+    /// - [`SentenceType::OLN`]
+    ///
+    /// ### Position
+    ///
+    /// - [`SentenceType::GLL`]
+    /// - [`SentenceType::DTM`]
+    ///
+    /// ### Radar
+    ///
+    /// - [`SentenceType::RSD`]
+    /// - [`SentenceType::TLL`]
+    /// - [`SentenceType::TTM`]
+    ///
+    /// ### Rudder
+    ///
+    /// - [`SentenceType::RSA`]
+    ///
+    /// ### Temperature
+    ///
+    /// - [`SentenceType::MTW`]
+    ///
+    /// ### Transit
+    ///
+    /// - [`SentenceType::GXA`]
+    /// - `SentenceType::RTF` (missing?!)
```

Comment:
@Dushistov @hargoniX seems like this type is missing from the Enum.
Can you point me in the right direction here - should we add it to the Enum or it's not implemented?